src/utils/hooks/useGetQuestionReviewHistory.tsx

```diff
@@ -62,6 +62,9 @@ const useGetQuestionReviewHistory = ({ isApplicant, ...variables }: UseGetQuesti
           title: strings.TITLE_HISTORY_SUBMITTED_BY_APPLICANT,
           // TODO translated message, that nothing is entered
           message: value?.text || '',
```

Comment:
It seems this is causing a small bug here. I noticed that there is a different between what is displayed in the SummaryView and what is displayed inside the ReviewHistory window:
![Screen Shot 2022-10-13 at 11 26 30 AM](https://user-images.githubusercontent.com/16461988/195459826-ea8b462c-93ca-460d-b6f7-b912d5d61d14.png)
![Screen Shot 2022-10-13 at 11 26 24 AM](https://user-images.githubusercontent.com/16461988/195459828-1d777386-d7ad-4ba6-9345-51689a5e2041.png)

Reply:
> It seems this is causing a small bug here. I noticed that there is a different between what is displayed in the SummaryView and what is displayed inside the ReviewHistory window:

Thanks for thorough testing. I suspected that not passing in a full "allResponses" object might cause problems. And I'm also slightly concerned it won't get the full data object it needs for certain evaluations.

So I think we need to be a bit careful with this.

I'll have another look at it soon, but I'm kinda worried that it might be difficult to get exactly right. One thing we might have to do -- and it's something I've thought would be useful for a while -- is add an additional "evaluated_parameters" field to application_response. Because as the application changes, dynamic parameters will have different values at different times, and therefore it's entirely possible to have incorrect values showing up in a Summary View component, as they get evaluated a new every time.

So what we would do is save the results of the *current* parameter evaluation along with each response, then every time that response is rendered going forward, it uses these pre-evaluated parameters rather than re-evaluating them.

But that's a bit more involved, so we should probably have a discussion about it before going there.

Reply:
Ok - I'm not fully understanding this. I suggest we have a meeting to discuss this later on today? Then I think we can create another issue to make an improvement on allResponses.

For now what do. you think about the possibility of having this object as optional and in this case nothing is passed and the response would only show things related to itself - not sure if this is possible though without having a look.

Reply:
> I suggest we have a meeting to discuss this later on today?

Sure, I'm good anytime before 4.

Reply:
Made a new issue to address this: #1394 

Will do that one first, then come back to this.

(I'll set this PR to Draft in the meantime)

Reply:
Sounds good - I noticed this PR is now based on the other. So I'll have a look on that one today - if time allows